src/operator/subgraph/dnnl/dnnl_transformer.cc

```diff
@@ -296,18 +368,33 @@ void SgDNNLSelfAttQKOp::Forward(const OpContext& ctx,
   DNNLStream::Get()->Submit();
 
   if (param_.quantized && !param_.enabled_float_output.has_value()) {
-    float* output_min = outputs[1].data().dptr<float>();
-    float* output_max = outputs[2].data().dptr<float>();
-
-    *output_min = min_output_;
-    *output_max = max_output_;
+    if constexpr (with_split) {
+      float* output_min = outputs[1].data().dptr<float>();
+      float* output_max = outputs[2].data().dptr<float>();
+      *output_min       = min_output_;
+      *output_max       = max_output_;
+    } else {
+      float* output_min_0 = outputs[2].data().dptr<float>();
+      float* output_max_0 = outputs[3].data().dptr<float>();
+      float* output_min_1 = outputs[4].data().dptr<float>();
+      float* output_max_1 = outputs[5].data().dptr<float>();
+      *output_min_0       = min_output_;
+      *output_max_0       = max_output_;
+      *output_min_1       = min_output_;
+      *output_max_1       = max_output_;
+    }
```

Comment:
There are only 3 output tensors. Doesn't it throw exception?

Reply:
it didn't but you are right, this code was incorrect